packages/frontend/src/App.tsx

```diff
@@ -52,6 +52,7 @@ const App = () => {
             }}>
             <Routes>
               <Route path='/' element={<LandingPage />} />
+              <Route path='/Feedback' element={<LandingPage />} />   // creating a new route for feedback page while still loading the landing page
```

Comment:
Neat, I didn't know comments like this were possible. I learned something new today 😊